dll.py

```diff
@@ -50,7 +88,15 @@ def append(self, val: T):
         self._size += 1
 
     def pop(self) -> T:
-        """Remove last node and return value"""
+        """
+        Removes and returns the tail of the list
+
+        If the list has more than one node, the tail is removed and the previous node of the tail is set as the new
```

Comment:
`the previous node of the tail` leest een beetje moeilijk, misschien gewoon `now last node`?

Reply:
Daar zat ik ook mee te stoeien...

Reply:
Ik vind het prima :)